xclim/ensembles.py

```diff
@@ -103,21 +103,21 @@ def ensemble_mean_std_max_min(ens: xr.Dataset) -> xr.Dataset:
     Examples
     --------
     >>> from xclim import ensembles
-    >>> import glob
-    >>> ncfiles = glob.glob('/*tas*.nc')
+    >>> from pathlib import Path
+    >>> ncfiles = Path().rglob('*tas*.nc')
     Create ensemble dataset
     >>> ens = ensembles.create_ensemble(ncfiles)
     Calculate ensemble statistics
     >>> ens_means_std = ensembles.ensemble_mean_std_max_min(ens)
     >>> print(ens_mean_std['tas_mean'])
     """
-    dsOut = ens.drop(ens.data_vars)
+    dsOut = ens._drop_vars(names=set(ens.data_vars))
```

Comment:
Question: Am I right in thinking that using drop in user code won't raise a warning, because it's a PendingDeprecationWarning which is disabled by default ? If so, I don't think we need to go this fast in removing `drop` statements. It forces us to use a private method (`_drop_vars`) which is not great practice. 

My suggestion would be to modify this PR to use the new methods `drop_vars`, and keep it on the back burner until the time xarray moves `drop` from PendingDeprecationWarning to DeprecationWarning. Then we make the switch and update the xarray minimum version.

Reply:
Sounds good. The changes here aren't urgent; I had about an hour yesterday and figured it would be good to get on top of some of the Deprecations coming our way. I imagine that once cftime v1.0.4.3 comes around, this will allow us to bump our versions up to xarray 0.14.3 where these deprecations will be relevant.